wire-ios/Wire-iOS/Sources/AppRootRouter.swift

```diff
@@ -421,21 +423,22 @@ extension AppRootRouter {
             return
         }
 
-        TrackingManager.shared.disableAnalyticsSharing = false
-        Analytics.shared.provider?.selfUser = selfUser
+        trackingManager.disableAnalyticsSharing = true
```

Comment:
Won't this override the user's stored preference every time the app is launched?

Reply:
Yep! Fixed it here: https://github.com/wireapp/wire-ios/commit/bd2750fe7733ccabe8628e32d9eb1806e67cc176